dotcom-rendering/src/amp/server/document.tsx

```diff
@@ -5,7 +5,7 @@ import createEmotionServer from '@emotion/server/create-instance';
 import createCache from '@emotion/cache';
 import he from 'he';
 
-import resetCSS from /* preval */ '@root/src/lib/reset-css';
+import { resets } from '@guardian/src-foundations/utils';
```

Comment:
What was `preval` doing before?

Reply:
Preval was used to read the file with `fs.readFileSync` and then spit it out into a string to be used a runtime

Reply:
Now it's in source it's already in a string